images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -157,6 +157,11 @@ function Get-LeiningenVersion {
     return "$(lein -v | Take-OutputPart -Part 0,1)"
 }
 
+function Get-MediainfoVersion {
+    $mediainfoVersion = mediainfo --version | Select-Object -Index 1 | Take-OutputPart -Part 2
```

Comment:
could you please change it to `mediainfoVersion = (mediainfo --version | Select-Object -Index 1 | Take-OutputPart -Part 2).Replace('v','')` to get rid of `v` in the output?
Currently, it looks like this
![image](https://user-images.githubusercontent.com/48208649/100979493-df9c0080-3554-11eb-8ecf-fdbf1422cb09.png)

Reply:
@miketimofeev fixed!

Also verified locally it works:
```
PS /home/vagrant> mediainfo --version
MediaInfo Command line,
MediaInfoLib - v19.09
PS /home/vagrant> (mediainfo --version | Select-Object -Index 1 | Take-OutputPart -Part 2).Replace('v', '')
19.09
```
Thank you!